app/views/articles/show.html.haml

```diff
@@ -16,7 +16,8 @@
           = f.submit "Like", class: "btn btn-block btn-primary"
   .row.p-5
     .col
-      = @article.body
+      %p.text-justify
+        = simple_format @article.body
```

Comment:
this would print %p tags when it's printing. why is this nested inside another paragraph tag